src/Microsoft.ML.FastTree/GamModelParameters.cs

```diff
@@ -300,12 +300,25 @@ public double[] GetFeatureBinUpperBounds(int featureIndex)
             return featureBins;
         }
 
+        /// <summary>
+        /// Get all the bin upper bounds.
+        /// </summary>
+        public double[][] GetBinUpperBounds()
```

Comment:
>GetBinUpperBounds [](start = 26, length = 17)

If this is a perf-critical method, then it would be quite slow given all the array alocations